config-templates/azure/azuredisk-csi-driver/1.23.0/deployment.yaml

```diff
@@ -0,0 +1,667 @@
+                - name: AZURE_CREDENTIAL_FILE
+                  valueFrom:
+                    configMapKeyRef:
+                      name: azure-cred-file
+                      key: path
+                      optional: true
+                - name: CSI_ENDPOINT
+                  value: unix:///csi/csi.sock
+                - name: KUBE_NODE_NAME
+                  valueFrom:
+                    fieldRef:
+                      apiVersion: v1
+                      fieldPath: spec.nodeName
+                - name: AZURE_GO_SDK_LOG_LEVEL
+                  value:
+              imagePullPolicy: IfNotPresent
+              securityContext:
+                privileged: true
+              volumeMounts:
+                - mountPath: /csi
+                  name: socket-dir
+                - mountPath: /var/data/kubelet/
+                  mountPropagation: Bidirectional
+                  name: mountpoint-dir
+                - mountPath: /etc/kubernetes/
+                  name: azure-cred
+                - mountPath: /dev
+                  name: device-dir
+                - mountPath: /sys/bus/scsi/devices
+                  name: sys-devices-dir
+                - mountPath: /sys/class/
+                  name: sys-class
+              resources:
+                limits:
+                  memory: 200Mi
+                requests:
+                  cpu: 10m
+                  memory: 20Mi
+          volumes:
+            - hostPath:
+                path: /var/data/kubelet/plugins/disk.csi.azure.com
+                type: DirectoryOrCreate
+              name: socket-dir
+            - hostPath:
+                path: /var/data/kubelet/
+                type: DirectoryOrCreate
+              name: mountpoint-dir
+            - hostPath:
+                path: /var/data/kubelet/plugins_registry/
+                type: DirectoryOrCreate
+              name: registration-dir
+            - hostPath:
+                path: /etc/kubernetes/
+                type: DirectoryOrCreate
+              name: azure-cred
+            - hostPath:
+                path: /dev
+                type: Directory
+              name: device-dir
+            - hostPath:
+                path: /sys/bus/scsi/devices
+                type: Directory
+              name: sys-devices-dir
+            - hostPath:
+                path: /sys/class/
+                type: Directory
+              name: sys-class
+  - kind: Deployment
+    apiVersion: apps/v1
+    metadata:
+      name: csi-azuredisk-controller
+      namespace: kube-system
+      labels:
+        app.kubernetes.io/name: azuredisk-csi-driver
+        app.kubernetes.io/version: 1.23.0
+    spec:
+      replicas: 2
+      selector:
+        matchLabels:
+          app: csi-azuredisk-controller
+      template:
+        metadata:
+          labels:
+            app.kubernetes.io/name: azuredisk-csi-driver
+            app.kubernetes.io/version: v1.23.0
+            app: csi-azuredisk-controller
```

Comment:
following annotation is missing. refer
https://github.com/IBM/ibm-satellite-storage/blob/master/config-templates/azure/azuredisk-csi-driver/1.18.0/deployment.yaml#L311